src/Run.ts

```diff
@@ -1,15 +1,55 @@
 import * as GeoJSON from 'geojson'
 import { ElevationProfile } from './ElevationProfile'
 import { FeatureType } from './FeatureType'
-import { Location } from './Location'
-import { SkiAreaFeature } from './SkiArea'
-import Source from './Source'
+import { SkiAreaSummaryFeature } from './SkiArea'
+import { Source } from './Source'
 import { Status } from './Status'
+import { exhaustiveMatchingGuard } from './util/exhaustiveMatchingGuard'
 
 export type RunGeometry = GeoJSON.LineString | GeoJSON.Polygon
 
+/**
+ * Represents a segment of a ski run/trail.
+ *
+ * Runs are derived from OpenStreetMap pistes, which are defined as all ways/areas/relations containing a "piste:type" tag.
+ * Piste relations support is limited, it is only used to enhance the data of its member ways. There is no way to access the relation itself in the output data.
+ *
+ * A single run as marked on the mountain may be broken up into several run features, especially if the run has branches, or changes in tags (e.g. difficulty, use).
+ *
+ * In the future, there might be a "Route" entity introduced to represent a whole run, which would be composed of multiple run features.
+ *
+ * Note: winter hiking trails and trails of other winter sports disciplines are also included in this dataset.
+ *
+ * In postprocessing:
+ * - overlapping ways are merged into a single run feature with multiple uses.
+ * - connected run segments with the same properties are merged into a single run feature.
```

Comment:
Nice. I figured OpenSkiMap merged runs, otherwise run counts would be much higher than those reported by ski areas themselves.